.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,39 @@
-# Description of the problem
+# Description of the issue
 
-- Explain the previous issue (description of the issue or the reference number of an open issue)
-- Explain the need for a new feature or component
-- Describe the added code
+- Describe the previous issue
+- Explain the need for a new feature, component or example
 
 # Description of the solution
 
 - Explain the solution to the existing issue
 - Explain the content of the new feature
 
-# How Has This Been Tested?
+# How has this been tested?
 
-- List and describe the test(s) for the added code (if there is no test for the pull request, explain why (i.e. the test already exists but was changed))
-
-- [ ] Test A
-- [ ] Test B
+- List and describe the new test(s) (i.e. what feature/parameter does it test)
+- Explain the changes and/or impacts on current tests
+- If not related to tests, how did you ensure that the solution works?
 
 # Documentation
 
-- If applicable, list the documentation files changed and their changes
-
-- [ ] File A
+- Summarize documentation
 
-# Future changes
+# Miscellaneous
 
+- Anything that you would like to add that does not fit into another category
 - If applicable, describe future changes or features that will be added in subsequent pull requests
 
-# Comments
+# Checklist (will be removed when merged)
+See [this page](https://chaos-polymtl.github.io/lethe/documentation/contributing.html#pull-requests) for more information about the pull request process.
+
+Code related list:
+- [ ] All in-code documentation related to this PR is updated (Doxygen format)
+- [ ] Lethe documentation is updated
+- [ ] If possible, new feature has a unit test or application test
+- [ ] Changelog (CHANGELOG.md) is updated
+- [ ] Code is indented with indent-all and .prm files with prm-indent
 
-- Anything that you want to add up that didn't fit in any other category can be written here
+Pull request related list:
+- [ ] Labels have been applied
+- [ ] There are at least 2 reviewers (or 1 if small feature) excluding the merger
```

Comment:
```suggestion
- [ ] There are at least 3 reviewers (or 2 if small feature) excluding the merger
```
Isn't it 2 except Bruno so 3, or do we keep 2 and accept that implicitly it means that Bruno does not count for one of them?
![image](https://github.com/chaos-polymtl/lethe/assets/107414376/dd63f715-7c0e-4fc5-9adb-2aec808c18ff)

Reply:
2 excluding one means 3 for me, am I wrong? 
By merger (the person that does the merge in Master) I mean Bruno or the person when Bruno is in vacations

Reply:
Ohh, you're right that's perfectly fine, I misread the last part sorry 😅

However, I think merger, should be replaced by "the person that merges the PR" (or some other alternative), because "merger" in the way you use it here is not really defined that way in the English language.

Reply:
Let's keep 2 reviewers for big features and 1 for small feature. Let'S just exclude me in this list. This way I can help in the end.